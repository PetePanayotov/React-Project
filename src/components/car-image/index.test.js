import React from 'react';
import {create , act} from 'react-test-renderer';
import CarImage from './index';


describe('CarImage Component' , () => {

    it('Should render catalog page car image' , () => {
        let root;

        act(() => {
            root = create(<CarImage page="catalog" imageUrl="https://google.com"/>)
        });

        expect(root.toJSON()).toMatchSnapshot();

    });

    it('Should render details page car image' , () => {
        let root;

        act(() => {
            root = create(<CarImage page="details" imageUrl="https://google.com"/>)
        });

        expect(root.toJSON()).toMatchSnapshot();

    });

});