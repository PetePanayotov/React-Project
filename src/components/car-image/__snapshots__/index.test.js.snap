// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CarImage Component Should render catalog page car image 1`] = `
<section
  className="catalog-imageWrapper"
>
  <img
    alt="Car Image"
    className="catalog-image"
    src="https://google.com"
  />
</section>
`;

exports[`CarImage Component Should render details page car image 1`] = `
<section
  className="details-imageWrapper"
>
  <img
    alt="Car Image"
    className="details-image"
    src="https://google.com"
  />
</section>
`;
